### PR TITLE
sql: fix some spelling errors

### PR DESCRIPTION
**Pull request by BramGruneir**
Release note: None

### CODE REVIEW

**Comment by BramGruneir** [comment]
Please make sure that the changes to `structured.proto` and thus `structured.pb.go` won't cause any issues.

**Comment by andreimatei** [comment]
:lgtm:

---

Review status: 0 of 7 files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20727#-:-L0LU9_cJ_IEmGgehOkF:bnfp4nl)*

**Review by jordanlewis — APPROVED** [review]
Thanks!

File: pkg/sql/sqlbase/table.go [hunk 1]
```diff
@@ -39,17 +39,17 @@ import (
...
+	"INT4":             ColumnType_INTEGER,
+	"INT8":             ColumnType_BIGINT,
+	"INT64":            ColumnType_BIGINT,
+	"BIT":              ColumnType_BIT,
+	"INT2":             ColumnType_SMALLINT,
+	"SMALLINT":         ColumnType_SMALLINT,
+	"FLOAT4":           ColumnType_REAL,
+	"REAL":             ColumnType_REAL,
+	"FLOAT8":           ColumnType_DOUBLE_PRECISION,
+	"DOUBLE PRECISION": ColumnType_DOUBLE_PRECISION,
```
Comment (jordanlewis) [inline pkg/sql/sqlbase/table.go]: Fairly sure this is fixing a real but minor bug - I think without this change users wouldn't be able to specify a DOUBLE PRECISION type with a precision type omdifier. Or something like that.

**Comment by m-schneider** [comment]
:lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20727#-:-L0LW0WZWruoVkNhDxRP:bnfp4nl)*

File: pkg/sql/sqlbase/table.go (hunk 1 above)
Comment (BramGruneir) [reply pkg/sql/sqlbase/table.go]: Looks like there's a test failure so you might be right. I'll update the release notes if this is the case.

Comment (BramGruneir) [reply pkg/sql/sqlbase/table.go]: Yep, was a minor bug, added the release note.

**Pull request merged**